features/storage/TESTS/blockdevice/general_block_device/main.cpp

```diff
@@ -48,6 +48,10 @@
 #include "FlashIAPBlockDevice.h"
 #endif
 
+#if COMPONENT_SDIO
+#include "SDIOBlockDevice.h"
```

Comment:
What happened if a targets adds component SDIO and SD and QSPIF ?

Reply:
All three blockdevices will be tested